frontend/src/app/main/ui/workspace/left_header.cljs

```diff
@@ -108,6 +112,25 @@
          {:class (stl/css :file-name)
           :title file-name
           :on-double-click start-editing-name}
+          ;;-- Persistende state widget
+         [:div {:class (case (mf/deref ref:persistence-status)
+                         :pending (stl/css :status-notification :pending-status)
+                         :saving (stl/css :status-notification :saving-status)
+                         :saved (stl/css :status-notification :saved-status)
+                         :error (stl/css :status-notification :error-status)
+                         (stl/css :status-notification))
+                :title (case (mf/deref ref:persistence-status)
```

Comment:
The same here

Reply:
Additionally you are derefing and watching the same value twice, which is also redundant operation